client/src/components/Register/Introduction.js

```diff
@@ -0,0 +1,100 @@
+/* global window */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const fallbacks = require('../../../lang/src/en.json');
+
+function Introduction({ canSkip, onContinue, onSkip }) {
+  const { ss: { i18n } } = window;
+
+  return (
+    <div>
+      <h2 className="mfa-section-title">
+        { i18n._t('MFAIntroduction.TITLE', fallbacks['MFAIntroduction.TITLE']) }
+      </h2>
+
+      <h4 className="mfa-feature-list-title">
+        { i18n._t('MFAIntroduction.HOW_IT_WORKS', fallbacks['MFAIntroduction.HOW_IT_WORKS']) }
+      </h4>
+
+      <ul className="mfa-feature-list">
+        <li className="mfa-feature-list-item">
+          <img
+            alt="Shields indicating additional protection"
+            aria-hidden="true"
+            className="mfa-feature-list-item__icon"
+            src="/resources/vendor/silverstripe/mfa/client/dist/images/extra-protection.svg"
+          />
+          <div>
+            <h5 className="mfa-block-heading mfa-feature-list-item__title">
+              { i18n._t(
+                'MFAIntroduction.EXTRA_LAYER_TITLE',
+                fallbacks['MFAIntroduction.EXTRA_LAYER_TITLE']
+              ) }
+            </h5>
+
+            <p className="mfa-feature-list-item__description">
+              { i18n._t(
+                'MFAIntroduction.EXTRA_LAYER_DESCRIPTION',
+                fallbacks['MFAIntroduction.EXTRA_LAYER_DESCRIPTION']
+              ) }
+              &nbsp;
+              <a href="#">{ i18n._t('MFA.FIND_OUT_MORE', fallbacks['MFA.FIND_OUT_MORE'])}</a>
```

Comment:
The href for this is available as a config prop in PHP. It's currently undefined, so perhaps we shouldn't render it until it has a value, but you can get this from PHP as well as the image URL

Reply:
I ended up defining another config var for this link, as I think it will be a broader docs link than the one that's currently there.